packages/mobile/src/escrow/EscrowedPaymentListItem.tsx

```diff
@@ -69,7 +69,8 @@ export class EscrowedPaymentListItem extends React.PureComponent<Props> {
 
   getDisplayName() {
     const { payment } = this.props
-    return payment.recipientContact ? payment.recipientContact.displayName : payment.recipientPhone
```

Comment:
the EscrowPayment recipientContact field was always undefined. And we shouldn't use it, we should be looking up the contact info in the recipient cache each time to get updates. Removing that field and its references to avoid confusion and opened https://github.com/celo-org/celo-monorepo/issues/2438